src/components/SecondaryPanes/UtilsBar.js

```diff
@@ -15,7 +14,7 @@ function debugBtn(onClick, type, className, tooltip, disabled = false) {
 
   return (
     <button className={classnames(type, className)} {...props}>
-      <Svg name={type} />
+      K?
```

Comment:
I was thinking that the button could be a simple `?` (since im used to it from other editors) with no K (since K is not necessarily meaningful).. not sure, what do you think?